python/src/ccf/ledger.py

```diff
@@ -517,8 +517,7 @@ def add_transaction(self, transaction):
         if COSE_SIGNATURE_TX_TABLE_NAME in tables:
             cose_signature_table = tables[COSE_SIGNATURE_TX_TABLE_NAME]
             cose_signature = cose_signature_table.get(WELL_KNOWN_SINGLETON_TABLE_KEY)
-            signature = json.loads(cose_signature)
-            cose_sign1 = base64.b64decode(signature["sig"])
+            cose_sign1 = base64.b64decode(cose_signature)
```

Comment:
Why base64?

Reply:
I think this's just how we serialise byte arrays in CCF

Reply:
https://github.com/microsoft/CCF/blob/main/include/ccf/ds/json.h#L69

Reply:
Perf is OK, leaving as is